upgrade-cockroach.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Upgrade Your Cluster's Version of CockroachDB
+summary: Learn how to upgrade your CockroachDB cluster to a new version.
+toc: false
+toc_not_nested: true
+---
+
+Because of CockroachDB's multi-active availability design, you can perform a "rolling upgrade" of CockroachDB on your cluster. This means you can upgrade individual nodes in your cluster one at a time without any downtime for your cluster.
+
+However, there are a few things you should keep in mind:
+
+  - For rolling upgrades to work, your cluster must be behind a load balancer or your clients must be configured to talk to multiple nodes. If your application communicates with a single node, bringing it down to upgrade its CockroachDB binary will cause your application to fail when the node goes offline.
+
+  - We recommend creating scripts to upgrade CockroachDB and not attempting to do so by hand.
+
+  - Only bring down only one node at a time, and wait at least one minute after it rejoins the cluster to bring down the next node to prevent CockroachDB from entering a failover state.
```

Comment:
nit: Delete second `only`, and I'd suggest restructuring the sentence a bit to start with the goal:

> To prevent CockroachDB from entering a failover state, bring down only one node at a time, and wait at least one minute after it rejoins the cluster before bringing down the next node.

Also, I'm not exactly sure what we mean by "failover state"...